src/buildstream/_frontend/widget.py

```diff
@@ -483,24 +488,23 @@ def print_heading(self, toplevel_project, stream, *, log_file):
         values["Maximum Build Tasks"] = context.sched_builders
         values["Maximum Push Tasks"] = context.sched_pushers
         values["Maximum Network Retries"] = context.sched_network_retries
+
+        if context.remote_cache_spec:
+            values["Cache Storage Service"] = format_spec(context.remote_cache_spec)
+
         text += self._format_values(values)
 
         if context.remote_execution_specs:
             specs = context.remote_execution_specs
 
-            def format_spec(spec):
-                if spec.instance_name:
-                    return "{} (instance: {})".format(spec.url, spec.instance_name)
-                return spec.url
-
             text += "\n"
             text += self.content_profile.fmt("Remote Execution Configuration\n", bold=True)
             values = OrderedDict()
             values["Execution Service"] = format_spec(specs.exec_spec)
-            values["Storage Service"] = format_spec(specs.storage_spec)
+            if specs.storage_spec:
```

Comment:
I think it would be better to have the summary redundantly specify the storage service, rather than make the user infer that is will be defaulted to the `storage-service`

Reply:
Done